x-pack/legacy/plugins/siem/public/components/timeline/body/events/stateful_event.tsx

```diff
@@ -43,11 +44,13 @@ interface Props {
   toggleColumn: (column: ColumnHeader) => void;
   updateNote: UpdateNote;
   width: number;
+  maxDelay: number;
```

Comment:
perhaps `maxDelay` could be an optional prop with a default of `0`?

Reply:
Good suggestion, adding this as optional and defaulting to zero for it. Tested it in the browsers and it looks to work out well.

To let you know, when it's zero its default behavior is as if there isn't a `maxDelay` and it will take as long as needed which is probably for the best. Otherwise, to turn on max "jank" and pretty disable this mechanism (other than an initial `setTimeout` effect) would be to set this to a value of `1`.